src/test/java/com/artipie/nuget/NugetITCase.java

```diff
@@ -43,6 +45,7 @@
  * @since 0.12
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@EnabledOnOs({OS.LINUX, OS.MAC})
```

Comment:
@genryxy probably you need to enable the test on windows only, not enable it unix-like systems

Reply:
@olegmoz in the previous commit it fails on windows https://github.com/artipie/artipie/runs/1197922355#step:5:7796

Reply:
@genryxy that's because `testcontainers` library does not support Windows. That is sad. It means you need to use a Linux image instead of Windows one

Reply:
@genryxy you may find such images here: https://hub.docker.com/_/microsoft-dotnet-core

Reply:
@olegmoz in this test `mcr.microsoft.com/dotnet/sdk:5.0` image is used